### ISSUE TITLE
Add Support for Specifying Catalogs to CLI

### ISSUE THREAD

**Issue by r-pascua** [HERA-Team/hera_sim#67]
Currently, foreground simulation using `hera_sim`'s CLI is restricted to the methods in the `foregrounds` module. In the future, we may want to allow for foregrounds to be specified by reference to catalogs. This idea should be revisited once the other simulators that are regularly used have been wrapped by `hera_sim`.

**Comment by steven-murray** [comment]
Yup, definitely agree with this. This should be high on the priority list once #33  is merged.

**Comment by steven-murray** [comment]
I think this is a duplicate of #94
